frontend/src/pages/CustomImages/ImageBuildDialog.js

```diff
@@ -148,10 +149,9 @@ export default function ImageBuildDialog(props) {
             <Input value={imageId} onChange={({ detail }) => {setImageId(detail.value)}} />
           </FormField>
         </div>
-        <textarea
-          spellCheck="false"
-          className="configuration-data" value={imageConfig}
-          onChange={(e) => {setState([...imageBuildPath, 'config'], e.target.value)}} />
+        {<ConfigView
+          value={imageConfig}
```

Comment:
It looks like the property for `ConfigView` is called `config` rather than `value` -- perhaps this should be changed?

Reply:
no `config` is fine. I'll change that

Reply:
done